### PR TITLE
Use AssertableSubscriber instead of subscribing to the Observable

### PR DESCRIPTION
**Pull request by VictorAlbertos**
Hi @kittinunf :)

I've modified RxFuelTest class in order to take advantage of the built-in support that RxJava offers to testing in a more "reactive" idiomatic way.

### CODE REVIEW

**Comment by kittinunf** [comment]
Nice! Thank you so much for this. I've been wanting to change it but have found a good time to work on it! Thanks!!!!

File: fuel-rxjava/src/test/kotlin/com/github/kittinunf/fuel/RxFuelTest.kt [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 import com.github.kittinunf.fuel.rx.rx_responseObject
 import com.github.kittinunf.fuel.rx.rx_responseString
 import com.github.kittinunf.fuel.rx.rx_string
 import org.hamcrest.CoreMatchers.notNullValue
-import org.hamcrest.CoreMatchers.nullValue
 import org.junit.Assert.assertThat
 import org.junit.Test
 import org.hamcrest.CoreMatchers.`is` as isEqualTo
 
+
```
Comment (kittinunf) [inline fuel-rxjava/src/test/kotlin/com/github/kittinunf/fuel/RxFuelTest.kt]: Can you remove this extra new lines? 🙏

**Comment by kittinunf** [comment]
Aside for one [nits]. This is good to merge 🚀 🚀 🚀 . Thank you!

**Comment by VictorAlbertos** [comment]
Done :)

**Comment by kittinunf** [comment]
Thank you!!!

**Pull request merged**